sources/packages/api/src/services/student-appeal/student-appeal.service.ts

```diff
@@ -140,6 +140,15 @@ export class StudentAppealService extends RecordDataModelService<StudentAppeal>
           );
         }),
       )
+      .orderBy(
+        `CASE 
+          WHEN EXISTS(${this.studentAppealRequestsService
```

Comment:
Did we try to do this based on the `status` directly?

Reply:
ya I tried with `status` it was not working